### DISCUSSION TITLE
iscsiadm / open-iscsi setup (for longhorn)

### DISCUSSION THREAD

**Discussion by exp0nge** [k3s-io/k3s#9987] [Q&A]
Longhorn requires iscsi to be setup, however, how is the test environment setup such this integration test can utilize it? 

https://github.com/k3s-io/k3s/pull/6445/files#diff-7d30a4acc21d7df52e37a1fb0f641a5f6bc5253d15cdc5d84449bd8f53433f1aR20

I would like to understand how to setup k3s + longhorn and I hit a similar issue that this test pre-checks but haven't been able to figure out what step is done.

This has become a little irrelevant for me as I discovered https://github.com/rancher/local-path-provisioner/pull/350 is in progress, but am curious to this question still

**Comment by brandond** [comment]
You just need to install the iscsi tools on your node before you can use LH. This is covered in the LH documentation, this is not a K3s question. https://longhorn.io/docs/1.6.1/deploy/install/#installing-open-iscsi
